### ISSUE TITLE
Excel files

### ISSUE THREAD

**Issue by cybermax** [zdavatz/covid19_ch#27]
Are there any Excel files with the data?

**Comment by zdavatz** [comment]
Yes, there is one here: https://www.bag.admin.ch/dam/bag/de/dokumente/mt/k-und-i/aktuelle-ausbrueche-pandemien/2019-nCoV/covid-19-datengrundlage-lagebericht.xlsx.download.xlsx/200325_Datengrundlage_Grafiken_COVID-19-Bericht.xlsx
